### PR TITLE
[RFR] Use arrow keys to play

### PR DESCRIPTION
**Pull request by xavierhans**
Add the lib curses to the project so we can play the game using the arrow keys.
Had to remove all the inputs. Add the option to quit and restart the game.

[Trello task 1](https://trello.com/c/xSNlmMDW/14-02-as-donald-i-want-to-restart-a-game-or-left-the-game)
[Trello task 2](https://trello.com/c/S2Pjwcsk/8-04-as-donald-i-want-to-play-only-using-the-arrow-keys-on-my-keyboard)
curses

### CODE REVIEW

File: src/board.py [hunk 1]
```diff
@@ -13,39 +15,56 @@ def create_empty_board():
...
+
+def convert_board_coord_in_screen_coord(board_coord, axis):
+    screen_coord = 4 if axis is X_AXIS else 2
+    screen_coord += 2 * board_coord if axis is X_AXIS else board_coord
+    if board_coord > 2:
+        screen_coord += 2 if axis is X_AXIS else 1
+    return screen_coord
+
+
+def display_cursor(cursor_y, cursor_x, board, current_action, stdscr):
```
Comment (zyhou) [inline src/board.py]: remove the multiple `elif`

```
if 1=1:
  // some code
  return 

if 2=2:
  // some code
  return 
```

File: src/board.py [hunk 2]
```diff
@@ -13,39 +15,56 @@ def create_empty_board():
...
+    elif current_action == ASKING_ROTATION:
+        for i in range(0, 3):
+            for j in range(0, 3):
+                y = convert_board_coord_in_screen_coord(cursor_y * QUADRANT_SIZE + i, Y_AXIS)
+                x = convert_board_coord_in_screen_coord(cursor_x * QUADRANT_SIZE + j, X_AXIS)
+                stdscr.addstr(y, x, board[cursor_y * QUADRANT_SIZE + i][cursor_x * QUADRANT_SIZE + j], curses.color_pair(2))
+    elif current_action == ASKING_ROTATION_DIRECTION:
+        clockwise_color = curses.color_pair(2)
+        anticlockwise_color = curses.color_pair(1)
+        if cursor_y:
```
Comment (zyhou) [inline src/board.py]: use ternary

File: src/game.py [hunk 3]
```diff
@@ -51,39 +53,131 @@ def is_the_game_a_draw(board, winners):
 def is_the_game_finished(board, check_draw):
```
Comment (zyhou) [inline src/game.py]: When function start with `is`, we're waiting to get a boolean value. Rename the function or add another

File: src/game.py [hunk 4]
```diff
@@ -51,39 +53,131 @@ def is_the_game_a_draw(board, winners):
...
+    stdscr.refresh()
+    # Start colors in curses
+    curses.start_color()
+    curses.init_pair(1, curses.COLOR_CYAN, curses.COLOR_BLACK)
+    curses.init_pair(2, curses.COLOR_RED, curses.COLOR_BLACK)
+    curses.init_pair(3, curses.COLOR_BLACK, curses.COLOR_WHITE)
+    curses.curs_set(0)
+
+
+def display_direction_for_the_player(current_action, end_game_msg, board, stdscr):
```
Comment (zyhou) [inline src/game.py]:
```
if end_game_msg:
        stdscr.addstr(PLAYER_SCREEN_OFFSET + 1, 2, end_game_msg, curses.color_pair(2))
       return

// other code
```

**Comment by fzaninotto** [comment]
Cruelly lacking unit tests. Try to unit test each of the methods / functions you create.

**Pull request merged**